### PR TITLE
Please review

### PR DESCRIPTION
**Pull request by danielEftimov**
@Mirakovska @jovan1ivanovski @eilievska

### CODE REVIEW

File: core/src/org/otw/open/dto/DrawablePixmap.scala [hunk 1]
```diff
@@ -52,5 +52,9 @@ class DrawablePixmap(val pixmapMask: Pixmap) extends Disposable {
 
   }
 
+  def changeTexture(texture: Texture) = {
```
Comment (eilievska) [inline core/src/org/otw/open/dto/DrawablePixmap.scala:55]: This method should probably be renamed and docs should be added.

**Comment by eilievska** [comment]
:+1:

**Comment by Mirakovska** [comment]
:+1:

**Comment by jovan1ivanovski** [comment]
:+1:

**Pull request merged**